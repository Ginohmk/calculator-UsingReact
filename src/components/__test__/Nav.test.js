/* eslint-disable comma-dangle */
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';

import renderer from 'react-test-renderer';
import NavBar from '../Nav';

describe('Test Nav Bar Component', () => {
  test('Testing Nav Bar ', () => {
    const tree = renderer
      .create(
        <Router>
          <NavBar />
        </Router>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
