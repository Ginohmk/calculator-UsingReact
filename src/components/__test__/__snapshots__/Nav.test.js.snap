// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Nav Bar Component Testing Nav Bar SnapSjot  1`] = `
<header
  className="header"
  data-testid="nav-data"
>
  <div>
    Math Magicians
  </div>
  <nav>
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
           Home 
        </a>
        <span>
          |
        </span>
      </li>
      <li>
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
        <span>
          |
        </span>
      </li>
      <li>
        <a
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
